### DISCUSSION TITLE
Multiple QueueEvents Instances

### DISCUSSION THREAD

**Discussion by johneke** [taskforcesh/bullmq#1263] [Q&A] [answered]
I have what I hope is a quick question: if I run `QueueEvents` instance on multiple servers (listening to the same queue), will all servers get all queue events? Or will only one server get the queue events?

Something like this:

```
# Server A

const queueEvents = new QueueEvents('Paint');

queueEvents.on('completed', ({ jobId }) => {
  console.log('done painting');
});
```

```
# Server B

const queueEvents = new QueueEvents('Paint');

queueEvents.on('completed', ({ jobId }) => {
  console.log('done painting');
});
```

Will both servers get all events? 

Thank you 🙏

**Comment by manast** [comment, marked as answer]
Yes.

**Comment by johneke** [comment]
Thank you! 🙇‍♂️
